dm/worker/worker_test.go

```diff
@@ -161,7 +162,8 @@ func (t *testServer) TestTaskAutoResume(c *C) {
 	// start task
 	cli := t.createClient(c, fmt.Sprintf("127.0.0.1:%d", port))
 	subtaskCfgBytes, err := ioutil.ReadFile("./subtask.toml")
-	_, err = cli.StartSubTask(context.Background(), &pb.StartSubTaskRequest{Task: string(subtaskCfgBytes)})
+	// strings.Replace is used here to uncomment extra-args to avoid mydumper connecting to DB and generating arg --tables-list which will cause failure
```

Comment:
is it the only way to handle this problem? it's tricky

Reply:
Yes, I admit that. Perhaps we can add failpoint to solve this problem?